client/packages/common/src/hooks/useUrlQuery/useUrlQuery.ts

```diff
@@ -18,6 +18,7 @@ export type UrlQueryValue =
 
 // CONSTANTS
 export const RANGE_SPLIT_CHAR = '_';
+export const NESTED_SPLIT_CHAR = '.';
```

Comment:
Btw we could also use some conventions from OpenAPI3?

https://swagger.io/docs/specification/describing-parameters/

For example, from the query params section:

```
Arrays can be serialized as:

    form – /products?color=blue,green,red or /products?color=blue&color=green, depending on the explode keyword
    spaceDelimited (same as collectionFormat: ssv in OpenAPI 2.0) – /products?color=blue%20green%20red
    pipeDelimited (same as collectionFormat: pipes in OpenAPI 2.0) – /products?color=blue|green|red

Objects can be serialized as:

    form – /points?color=R,100,G,200,B,150 or /points?R=100&G=200&B=150, depending on the explode keyword
    deepObject – /points?color[R]=100&color[G]=200&color[B]=150

```


For example, `sensor[name]` instead of `sensor.name` or arrays could be used for ranges: `datetime=2023-11-01+00%3A00,2023-11-23+00%3A00`).

Tagging @mark-prins as well.